akka-cluster/src/multi-jvm/scala/akka/cluster/StressSpec.scala

```diff
@@ -551,14 +551,15 @@ abstract class StressSpec
     }
     enterBarrier("result-aggregator-created-" + step)
     runOn(roles.take(nbrUsedRoles): _*) {
-      val resultAggregator = clusterResultAggregator
+      val resultAggregator = identifyClusterResultAggregator()
       phiObserver ! ReportTo(resultAggregator)
       statsObserver ! Reset
       statsObserver ! ReportTo(resultAggregator)
     }
+    enterBarrier("result-aggregator-identified-" + step)
```

Comment:
without this barrier another node can continue and establish the blackhole, which will cause the timeout for the identify